azerothcore/azerothcore-wotlk
# fix(DB/Loot): removes Elixir of Greater Agility from creature loot table

## Changes Proposed:
-  Removes Elixir of Greater Agility from creature loot table. Specifically from these mobs GUIDs 7438, 9416 and 11053.

## Issues Addressed:

- Closes #7905 
- Closes https://github.com/chromiecraft/chromiecraft/issues/1739

## SOURCE:


## Tests Performed:

- As below

## How to Test the Changes:


1. Run SQL command and check that it changes 3 rows
2. Check the creature loot table for the item
     * ``SELECT * FROM `creature_loot_template` WHERE `Item` = 9187;``
     * Should be 0 mobs that drop it

## Known Issues and TODO List:



## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
